### ISSUE TITLE
HTML table colspan attribute are being read by screenreaders

### ISSUE THREAD

**Issue by tvarshav** [FreedomScientific/VFO-standards-support#599]
### Summary

>We are using Time card application configured in Oracle HCM. Visual impairment employees are not able to enter their time because screenreaders are reading out wrong date label  (Wed. in stead of Monday for example) in all the browsers.

### Expected result
HTML table/column headers should be recognized and read out properly by the screenreaders (JAWsincluding)

### Actual result
Screen readers are reading wrong dates after Sunday (Sunday is the only title read correctly, and then it follows by Wed instead of Monday). 
Below are the results after investigation. 
As per the note from the Oracle Accessibility Engineer:
1.	We’ve taken JPMC code, analyzed it with JAWS 2021 and Chrome Version 96.0.4664.110 (Official Build) (64-bit)
	Both their code and our code works as intended , with editable fields providing the right column header when a person is editing a data table cell related to the column.
2.	We’ve taken JPMC code, analyzed it with OATB , specifically reviewing table header / column, colspan, data cell interactions , etc. 
	The Oracle Accessibility Toolbar, (OATB), is based on the AxeCore ruleset that Deque provides for accessibility inspections within the browser.
3.	We’ve analyzed Oracle code against their code on the same table structure.
We determined that both code bases are pulling from the correct column header / column span and row information internally when reviewing on our test environments as well as locally testing what they have given us from a screen grab of the original issue .
Oracle recommendation was to raise here because they believe it is similar to https://github.com/FreedomScientific/VFO-standards-support/issues/545 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

**Comment by tvarshav** [comment]
Additional Information:
JAWS version and build number it occurs in all the version we tried with 2020.xxx through 2021
Operating System and version Windows 10
Browser and version: Chrome 94.x and up, FireFox 86
